app/domain/authentication/authn_jwt/validate_request_body.rb

```diff
@@ -0,0 +1,40 @@
+require 'command_class'
+
+# This class validates that request body contains JWT encoded token
+module Authentication
+  module AuthnJwt
+
+    ValidateRequestBody ||= CommandClass.new(
+      dependencies: {
+      },
```

Comment:
a command class without any dependencies should raise the question of its existence. do we need this class? can we move the logic of this class to its calling class? 

we may have a redundant layer here that makes the code reader make an extra hop between classes.

Reply:
I'll convert it to regular class